test/Fixtures/CustomerFixtureTrait.php

```diff
@@ -159,6 +169,7 @@ public function getShippingAddress(): Address
             ->setZip('60386')
             ->setCity('Frankfurt am Main')
             ->setCountry('DE')
-            ->setState('DE-BO');
+            ->setState('DE-BO')
+            ->setShippingType(ShippingTypes::EQUALS_BILLING);
```

Comment:
Old tests should not be changed.
This parameter is present for upl invoice only

Reply:
but you wont see if its absence would cause an error

Reply:
there are two types of maximum customers now, since upl is a case that does not seemlesly fit into the rest of the papi